File: ios/RNMBX/RNMBXRasterLayerViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXRasterLayer, RNMBXRasterLayerViewManager, RCTViewManager)

// symbol layer props
RCT_EXPORT_VIEW_PROPERTY(sourceLayerID, NSString)

// standard layer props
#include "CommonLayerProperties.H"

@end

File: ios/RNMBX/RNMBXRasterLayerViewManager.swift
@objc(RNMBXRasterLayerViewManager)
class RNMBXRasterLayerViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return false
    }
  
    override func view() -> UIView! {
      let layer = RNMBXRasterLayer()
      layer.bridge = self.bridge
      return layer
    }
}
